Makefile

```diff
@@ -207,6 +216,12 @@ target/bundle/admin: resources/public/admin
 	mkdir -p target/bundle
 	cp -r resources/public/admin target/bundle/admin
 
+# Include version file
+target/bundle/version:
+	mkdir -p target/bundle
+	make git-details.json
+	mv git-details.json target/bundle/git-details.json
+
```

Comment:
I don't get why it isn't just a single make target like `target/bundle/git-details.json`

Reply:
I wanted to keep it convenient for dev  to just move into `dev/resources`

Reply:
ended up moving away from make